app/assets/stylesheets/views/_organisations.scss

```diff
@@ -110,6 +110,10 @@
   border-top-style: solid;
 }
 
+.organisations__section {
```

Comment:
Not sure this class is necessary, but this is partly because I haven't got my changes merged in yet. I've been using the `organisations__margin-bottom` class (below) to wrap each section of the page. Let's leave it for now and we'll consolidate this when we get everything merged.

Reply:
I've made a note to revisit this 👍